python/tests/README.md

```diff
@@ -10,6 +10,15 @@ can be run against a live Nessie engine.
    `./mvnw quarkus:dev -am -pl :nessie-quarkus` locally.
 1. Run `pytest --record-mode=all tests/` (in a venv with `requirements_dev.txt` installed)
    to add test data for newly added tests and updated tests data.
+   * Note: when running `test_nessie_auth.py` in recording more, the tests will most likely fail out-of-the box
+     because the server will not recognize the auth tokens those tests use. In order to record cassettes for the
+     authentication tests:
+     1. Follow the instructions in the [Nessie Server README.md file](../../servers/quarkus-server/README.md) to start
+        the server in test mode with a real authentication provider and generate a real auth token.
+     1. Put real tokens into test sources
+     1. Run tests in recording mode
```

Comment:
Run auth tests in recording mode, such as `pytest --record-mode=all tests/test_nessie_cli_auth.py`

Reply:
updated: a18b0aff916541eff14766b60b9dfdede0270d2f